docs/user-guide/getting-started/running.md

```diff
@@ -1,14 +1,80 @@
-Running the Certora Prover
-==========================
+# Running the Certora Prover
 
-```{todo}
-This documentation is incomplete.  In the meantime,
-see [Lesson 2 of the tutorial][lesson]
-for a detailed walkthrough of how to write a basic property, verify it using
-the Certora Prover, and understand the verification results.
+The `certoraRun` utility simplifies the verification process by invoking the Solidity compiler and then sending the job to Certora’s servers.
 
-Feel free to post questions at [discord help-desk channel](https://discord.com/invite/Z9wQfRqZYY).
+The most commonly used command is:
+
+```bash
+certoraRun contractFile:contractName --verify contractName:specFile
+```
+
+If `contractFile` is named `contractName.sol`, the command can be further simplified to:
+
+```bash
+certoraRun contractFile --verify contractName:specFile
+```
+
+A concise summary of these options can be viewed by using:
+
+```bash
+certoraRun --help
+```
+
+## Usage Example with ERC20 Contract
+
+To demonstrate the usage, let's consider an ERC20 contract named `ERC20.sol` from the [Certora Tutorials Repository](https://github.com/Certora/tutorials-code/blob/master/lesson2_started/erc20/ERC20.sol). The corresponding spec file is named `ERC20.spec`:
+
+```cvl
+/** @title Transfer must move `amount` tokens from the caller's
+ *  account to `recipient`
+ */
+rule transferSpec(address recipient, uint amount) {
+
+    env e;
+    
+    // `mathint` is a type that represents an integer of any size
+    mathint balance_sender_before = balanceOf(e.msg.sender);
+    mathint balance_recip_before = balanceOf(recipient);
+
+    transfer(e, recipient, amount);
+
+    mathint balance_sender_after = balanceOf(e.msg.sender);
+    mathint balance_recip_after = balanceOf(recipient);
+
+    // Operations on mathints can never overflow nor underflow
+    assert balance_sender_after == balance_sender_before - amount,
+        "transfer must decrease sender's balance by amount";
+
+    assert balance_recip_after == balance_recip_before + amount,
+        "transfer must increase recipient's balance by amount";
+}
+``` 
+
+You can run the Certora Prover with the following command:
+
+```bash
+certoraRun ERC20.sol --verify ERC20:ERC20.spec --solc solc8.0
 ```
 
+This command triggers a verification run on the `ERC20` contract from the solidity file `ERC20.sol`, checking all rules in the specification file `ERC20.spec`. The `--solc` option specifies the version of the Solidity compiler to be used (in this case, version 0.8.0).
+
+## Results
+
+While running, the Prover will print various information to the console about the run. In the end, the output will look similar to this:
+
+```text
+...
+[INFO]: Process returned with 100
+
+Job is completed! View the results at [Certora Prover Platform](https://prover.certora.com/)
+
+Finished verification request
+ERROR: Prover found violations:
+ERROR: [rule] transferSpec
```

Comment:
```suggestion
...
Job is completed! View the results at [Certora Prover Platform](https://prover.certora.com/)

Finished verification request
ERROR: Prover found violations:
ERROR: [rule] transferSpec
```

Reply:
btw I'm pretty sure our output is more specific today, so I'd use an actual run on this code + rule with a proper link that users can click here